src/types/customMarker.ts

```diff
@@ -0,0 +1,10 @@
+export interface MarkerCustomAttrs {
```

Comment:
hmm, any specific reason why defining this as interface and not a type?

Reply:
Not really. Do you think it should be a type instead? In my mind interfaces are essentially a means of describing the shapes of data, like an object. I am not sure if this should be a type or an interface to be honest. Still trying to figure out the differences between those two.

Reply:
I'm also trying to understand them fully. So far, I'm trying to use types where possible because they are more strict, you cannot re-declare them. Intefaces on the other hand can be "merged" - if you declare them twice in your code it will accept the merged structure when compiling the code (ref https://www.typescriptlang.org/docs/handbook/declaration-merging.html#merging-interfaces)

I've found this article a while back and I like how it describes the differences - https://www.frontendeng.dev/blog/7-everything-you-should-know-about-custom-types-in-typescript#advantages-of-using-types-in-typescript

Happy to discuss this offline and decide on an approach to follow for the app as a whole

Reply:
It will be converted to type.